src/pages/cart-drawer/drawer-cart-box-item/drawer-cart-box-item-content-box.tsx

```diff
@@ -0,0 +1,41 @@
+import { FC } from "react";
+import {
+  ContentBox,
+  PriceBox,
+  ProductImg,
+  TypographyDiscountPrice,
+  TypographyPrice,
+  TypographyProductName,
+} from "./drawer-cart-box-item-styled";
+import { Box } from "@mui/material";
+import { CartProductModel } from "../drawer-cart-box/drawer-cart-box";
+
+interface ContentBoxInterface {
+  cartItem: CartProductModel;
+}
+
+const DrawerCartBoxItemContentBox: FC<ContentBoxInterface> = ({ cartItem }) => {
+  return (
+    <ContentBox>
+      <ProductImg />
+      <Box sx={{ width: "100%" }}>
+        <TypographyProductName>{cartItem.product.name}</TypographyProductName>
+        <PriceBox>
+          {cartItem.product.discountPrice == null ? null : (
+            <TypographyDiscountPrice>
+              {`${cartItem.product.regularPrice}zł`}
+            </TypographyDiscountPrice>
+          )}
+
+          {cartItem.product.discountPrice == null ? (
```

Comment:
You can use short form like `cartItem.product.discountPrice ? ...` no need to check null explicitly